https://github.com/gwsw/less/issues/344

--- a/Makefile.in
+++ b/Makefile.in
@@ -16,9 +16,6 @@ LDFLAGS = @LDFLAGS@
 CPPFLAGS = @CPPFLAGS@
 EXEEXT = @EXEEXT@
 O=o
-ifneq ($(strip $(LESSTEST)),)
-CPPFLAGS += -DLESSTEST
-endif
 
 LIBS = @LIBS@
 
@@ -99,11 +96,8 @@ uninstall:
 info:
 install-info:
 dvi:
-installcheck:
-
 check:
-	if [ -x less$(EXEEXT) ] && ./less$(EXEEXT) -V | grep -q LESSTEST; then :; else ${MAKE} clean; ${MAKE} LESSTEST=1; fi
-	objdir=$$(pwd); rflags=""; if [ -z "${VERBOSE}" ]; then rflags=-e; fi; cd $(srcdir)/lesstest && ${MAKE} && ./runtest $$rflags -l "$$objdir/less" lt
+installcheck:
 
 TAGS:
 	cd ${srcdir} && etags *.c *.h
@@ -132,3 +126,4 @@ distclean: clean
 
 realclean: distclean
 	rm -f TAGS
+
-- 
2.39.2

